.github/workflows/prune-uploads-dev.yml

```diff
@@ -12,10 +12,10 @@ jobs:
         working-directory: data-serving/scripts/prune-uploads
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
```

Comment:
Better to go straight to 3.10?

Reply:
3.9 is not EOL until [2025](https://devguide.python.org/#status-of-python-branches) so no rush on this.